app/src/main/java/com/lagradost/quicknovel/providers/RoyalRoadProvider.kt

```diff
@@ -375,6 +375,18 @@ class RoyalRoadProvider : MainAPI() {
     override suspend fun loadHtml(url: String): String? {
         val response = app.get(url)
         val document = Jsoup.parse(response.text)
-        return document.selectFirst("div.chapter-content")?.html()
+        val styles = document.select("style")
+        val hiddenRegex = Regex("^\\s*(\\..*)\\s*\\{")
+        val chap = document.selectFirst("div.chapter-content")
+        styles.forEach { style ->
+            hiddenRegex.findAll(style.toString()).forEach {
```

Comment:
Ideally, this would be `style.text()`, but that gives an empty string/null and results in the whole thing just doing nothing